books/free-programming-books-de.md

```diff
@@ -150,9 +152,14 @@
 * [React lernen und verstehen](https://lernen.react-js.dev) - Manuel Bieh (HTML)
 
 
+### Joomla
+
+* [Joomla! 3 - Das umfassende Handbuch](https://openbook.rheinwerk-verlag.de/joomla_3/) - Richard Eisenmenger
+
+
 ### LaTeX
 
-* [LaTeX - eine Einführung und ein bißchen mehr...](http://www.fernuni-hagen.de/imperia/md/content/zmi_2010/a026_latex_einf.pdf) - Manuela Jürgens, Thomas Feuerstack (PDF)
+* [LaTeX - eine Einführung und ein bisschen mehr ...](http://www.fernuni-hagen.de/imperia/md/content/zmi_2010/a026_latex_einf.pdf) - Manuela Jürgens, Thomas Feuerstack (PDF)
```

Comment:
Why this edit?

Reply:
The cover is clear :wink:...

![image](https://user-images.githubusercontent.com/3125580/193533561-695e30b4-feab-4c82-8d9a-b2cd1c10724b.png)

Reply:
Yup. At first I was also afraid, this could be old german spelling, but the cover makes it clear :D